en/02_Developer_Guides/01_Templates/01_Syntax.md

```diff
@@ -69,32 +68,33 @@ This will result in trying to fetch a `Title` property from the data being displ
 Variables can be chained together, and include arguments.
 
 ```ss
-$Foo(param)
+$Foo("param")
 $Foo.Bar
 ```
 
 These variables will call a method / field on the object and insert the returned value as a string into the template.
 
+Arguments don't have to be literals - you can pass template variables as arguments as well.
+
 > [!WARNING]
-> If you wish to pass arguments to getter functions, you must use the full method name. e.g. `$Thing` will try to access `Thing` as a property, which will ultimately result in `getThing()` being called with no arguments To pass arguments you must use `$getThing('param')`.
->
-> Also, arguments must be literals, and cannot be other template variables (`$getThing($variable)` will pass the literal string `'$variable'` to the `getThing()` method).
+> Arguments cannot be literal arrays (e.g `$myMethod(["val1", "val2"])`). The behaviour for this is currently undefined.
 
 - `$Foo("param")` will call `$obj->Foo("param")`
 - `$Foo.Bar` will call `$obj->Foo()->Bar`
+- `$Foo($Bar)` will call `$obj->Bar` and then pass the result into `$obj->Foo($result)`
 
 > [!NOTE]
-> Arguments passed into methods can be any non-array literal type (not just strings), e.g:
+> Arguments passed into methods can be almost any type (not just strings), e.g:
```

Comment:
At first glance you might think to leave as-is cause we still can't do arrays - but it doesn't have to be a literal anymore which the old text said it does.